ethereum-consensus/src/phase0/block_processing.rs

```diff
@@ -309,7 +315,96 @@ pub fn get_validator_from_deposit(deposit: &Deposit, context: &Context) -> Valid
     }
 }
 
-pub(crate) const DEPOSIT_MERKLE_DEPTH: usize = DEPOSIT_CONTRACT_TREE_DEPTH + 1;
+pub const DEPOSIT_MERKLE_DEPTH: usize = DEPOSIT_CONTRACT_TREE_DEPTH + 1;
```

Comment:
is there a reason visibility changed here?

Reply:
it was not being included in the subsequent `spec` modules without that change.

Reply:
lets just get rid of this, https://github.com/ralexstokes/ethereum-consensus/pull/379